c7n/resources/vpc.py

```diff
@@ -2221,3 +2222,12 @@ def process(self, resources):
                     e.response['Error']['Message'])
             else:
                 raise
+
+    def process_log_group(self, logroup):
+        client = local_session(self.manager.session_factory).client('logs')
+        lg = client.describe_log_groups(logGroupNamePrefix=logroup)
+        if lg.get('logGroups') == []:
+            try:
+                client.create_log_group(logGroupName=logroup)
+            except client.exceptions.InvalidParameterException:
```

Comment:
You can probably omit this try/except block since the handler isn't doing anything with it besides raising

Reply:
Hey, thanks for the review! The reason I put a `raise` here, is to stop the execution of the program when `InvalidParameterException` pops up , else it goes ahead and also makes the create_flow_logs api call which is not intended

Reply:
Other exceptions will still bubble up here. Try this out locally to see what I am saying:

```python
class InvalidParameterException(Exception):
   pass

try:
   raise InvalidParameterException
except InvalidParameterException:
   print('caught it')

try:
   raise Exception()
except InvalidParameterException:
   print('didn\'t catch it')
```